src/vm/threads.h

```diff
@@ -3721,7 +3721,7 @@ class Thread: public IUnknown
     ARG_SLOT CallPropertyGet(BinderMethodID id, OBJECTREF pObject);
     ARG_SLOT CallPropertySet(BinderMethodID id, OBJECTREF pObject, OBJECTREF pValue);
 
-#if defined(FEATURE_HIJACK) && !defined(PLATFORM_UNIX)
+#if defined(FEATURE_HIJACK) && !defined(FEATURE_PAL)
```

Comment:
@parjong I would actually like to replace the usage of `FEATURE_HIJACK && !PLATFORM_UNIX` in VM later by what you did, but I want to do it everywhere as a single separate change.  I would like the PLATFORM_UNIX to be used only to control things that are not explicitly related to PAL (like e.g. slash vs backslash etc) and use FEATURE_PAL where the stuff depends on PAL (like the hijacking on Unix), but is not necessarily Unix specific (if we ever added support for some non-Unix OS in the future).
So I'd prefer keeping using FEATURE_PAL in the excepx86.cpp.